src/Congo.WebApi/Congo.WebApi.csproj

```diff
@@ -5,8 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.10" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.10" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.9" />
```

Comment:
Why are you downgrading the SqlServer version?

Reply:
I never touched that I honestly dont know how that got into my pr someone else must have changed it when I pulled dev into my branch